_posts/2019-08-25-License.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
Can this please stay in the root directory as LICENSE.md ? That's the Git/GitHub conventional location for a license file, and other pages in the `guide` directory can uplink to it.